src/AddTodo.js

```diff
@@ -0,0 +1,31 @@
+import {useState} from "react";
+
+export default function AddTodo({handleAddTask}) {
```

Comment:
Have a read about the module pattern and export the class at the end of the file i.e

```
const AddTodo = () => {
   // your code
}

export default AddTodo;
```

Also please user arrow functions instead of the function.